.gitignore

```diff
@@ -8,6 +8,8 @@ local*.py
 .pydevproject
 .DS_Store
 
+node_modules
+
```

Comment:
Shouldn't be part of this PR

Reply:
This PR introduces eslint node_package.
Adding packages without this would be a really bad idea.